### DISCUSSION TITLE
Integration with Lightline

### DISCUSSION THREAD

**Discussion by dr-costas** [gruvbox-community/gruvbox#190] [Q&A]
Hi, 

How can I integrate Gruvbox with Lightline?

**Comment by rbong** [comment]
Lightline themes can be defined a couple different ways:

Either lightline itself can define an autoload file in [`autoload/lightline/colorscheme/<theme>.vim`](https://github.com/itchyny/lightline.vim/tree/master/autoload/lightline/colorscheme) or the colorscheme plugin itself can define that same file. Check out any of the existing colorscheme files for an example.

Gruvbox [does have this file](https://github.com/gruvbox-community/gruvbox/blob/master/autoload/lightline/colorscheme/gruvbox.vim), so you should be able to set the file like this:

```vim
let g:lightline = {
      \ 'colorscheme': 'gruvbox',
      \ }
```

Let me know if there are any issues.

If you would like to customize the theme, just create a file in your runtime path (ex. `~/.vim`) named something like `autoload/lightline/colorscheme/custom_gruvbox.vim`, copy the existing theme, and make any changes you'd like to see. Consider contributing a PR if you've made improvements to the theme that others can benefit from.

**Comment by dr-costas** [comment]
I tried to so what you are saying but it did not worked.

**Comment by dr-costas** [comment]
This is what I'm getting:

![Uploading Screenshot 2023-09-12 at 9.02.45.png…]()

**Reply by rbong** [reply]
The screenshot is broken for me.

**Comment by dr-costas** [comment]
I even tried copying the `gruvbox.vim` file at the `ligthline.vim/autoload/lightline/colorscheme` directory, but still the same.

**Comment by rbong** [comment]
Please post your `.vimrc`, `init.vim`, or `init.lua` (depending on which you use) that you're having this issue with.

For me, this `.vimrc` works:

```vim
set background=dark
set laststatus=2

call plug#begin()
  Plug 'gruvbox-community/gruvbox'
  Plug 'itchyny/lightline.vim'

  let g:lightline = {
        \ 'colorscheme': 'gruvbox',
        \ }
call plug#end()

colorscheme gruvbox
```

Please also make sure you're using the latest version of lightline and gruvbox.
